From 362b8a94c0ffaa63af3a5a5772c29303be009640 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fabiano=20Fid=C3=AAncio?= <fidencio@redhat.com>
Date: Mon, 28 Aug 2017 13:17:49 +0200
Subject: [PATCH 78/93] SDAP: Add a debug message to explain why a backend was
 marked offline
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This new debug message may help us when debugging the cases where a
backend was marked offline but it shouldn't be.

Related: https://pagure.io/SSSD/sssd/issue/2976

Signed-off-by: Fabiano Fidêncio <fidencio@redhat.com>

Reviewed-by: Jakub Hrozek <jhrozek@redhat.com>
---
 src/providers/ldap/sdap_id_op.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/providers/ldap/sdap_id_op.c b/src/providers/ldap/sdap_id_op.c
index 3a3de3643711fc2e604b876f401a88b486f941d5..85622663598d3091f735d56b5c81f9b36506a085 100644
--- a/src/providers/ldap/sdap_id_op.c
+++ b/src/providers/ldap/sdap_id_op.c
@@ -608,6 +608,10 @@ static void sdap_id_op_connect_done(struct tevent_req *subreq)
 
             default:
                 /* do not attempt to retry on errors like ENOMEM */
+                DEBUG(SSSDBG_TRACE_FUNC,
+                      "Marking the backend \"%s\" offline [%d]: %s\n",
+                      conn_cache->id_conn->id_ctx->be->domain->name,
+                      ret, sss_strerror(ret));
                 can_retry = false;
                 is_offline = true;
                 be_mark_offline(conn_cache->id_conn->id_ctx->be);
-- 
2.14.1

